### PR TITLE
Get started Update Revision 2

### PR DESCRIPTION
**Pull request by elehmer**
These are the updates that Nancy requested: Fixed table items, changed external URLs to open new tab. I also fixed the indenting issue after the lists in `climate_change_and_climate_data_in_california.md` by using a markdown comment to end the list explicitly. The ordered list still displays as an unordered list and the blockquote still needs restyling.

### CODE REVIEW

File: content/get-started/climate-change-and-climate-data-in-california.md [hunk 1]
```diff
@@ -44,18 +44,22 @@ The 2020 Adaptation Planning Guide (APG) provides guidance to local governments
...
+- <a href="https://resilientca.org/apg/" target="_blank">2020 Adaptation Planning Guide</a>
+- <a href="https://www.opc.ca.gov/webmaster/ftp/pdf/agenda_items/20180314/Item3_Exhibit-A_OPC_SLR_Guidance-rd3.pdf" target="_blank">State of California Sea Level Rise Guidance, 2018 Update</a>
+- <a href="http://opr.ca.gov/docs/20180313-Building_a_Resilient_CA.pdf" target="_blank">Planning and Investing for a Resilient California: A Guidebook for State Agencies</a>
+- <a href="https://opr.ca.gov/planning/general-plan/" target="_blank">General Plan Guidelines and Technical Advisories</a> (See Chapter 4, Safety Element, and Chapter 8 for specific guidance on addressing climate change).
+- <a href="http://www.opr.ca.gov/docs/20200720-Vulnerable_Communities.pdf" target="_blank">Defining Vulnerable Communities in the Context of Climate Change</a>
+- Various state agencies produce regional, topical, and impact-specific guidance documents and technical resources that are compiled and highlighted on the State Adaptation Clearinghouse as Featured Resources, organized under <a href="https://resilientca.org/topics/" target="_blank">Topic Areas</a>.
+
+[//]: # (Using Comment Trick to End List)
```
Comment (clhenrick) [inline content/get-started/climate-change-and-climate-data-in-california.md:63]: Neat trick @elehmer. I think the alignment issues may have been caused by a space on the line between the last list item and the next paragraph. Either way this seems to fix it.

**Review by clhenrick — APPROVED** [review]
Thanks @elehmer LGTM

File: content/get-started/accessing-data-on-cal-adapt.md [hunk 2]
```diff
@@ -16,14 +16,14 @@ There are many ways to access, visualize, and download data from Cal-Adapt:
...
  
-*Cal-Adapt file server*: We store climate data provided by California’s scientific and research community in their original formats on the [Cal-Adapt file server](https://albers.cnr.berkeley.edu/data/). You can download NetCDF files of gridded datasets and CSV files of station data from the server. Links to individual datasets are also included in the [data catalog](/data/download/).
+*Cal-Adapt file server*: We store climate data provided by California’s scientific and research community in their original formats on the <a href="https://albers.cnr.berkeley.edu/data/" target="_blank">Cal-Adapt file server</a>. You can download NetCDF files of gridded datasets and CSV files of station data from the server. Links to individual datasets are also included in the [data catalog](/data/download/).
```
Comment (mukhtyar) [inline content/get-started/accessing-data-on-cal-adapt.md:19]: Replace all instances of `data-catalog` with `Data Download Tool`. I did that in one of my recent PR's so maybe you won't need to.

Comment (elehmer) [reply content/get-started/accessing-data-on-cal-adapt.md:19]: Yeah I was wondering why Nancy was using that language. Ok I'll fix.

**Pull request merged**